### PR TITLE
Refactor Schnelleinsteig third round

### PR DESCRIPTION
**Pull request by Thrillberg**

### CODE REVIEW

File: imperial.js [hunk 1]
```diff
@@ -246,6 +255,21 @@ export default class Imperial {
...
+  handleTaxation(action) {
+    if (action.payload.slot === "taxation") {
+      const nationName = action.payload.nation;
+      const nation = this.nations.get(nationName);
+      const taxes =
+        this.factoryCount(nationName) * 2 +
+        this.flagCount(nationName) -
+        this.unitCount(nationName);
+      nation.treasury += taxes;
+      nation.taxChartPosition = "6";
```
Comment (jreut) [inline imperial.js:267]: 🧐

File: schnelleinsteig.test.js [hunk 2]
```diff
@@ -756,54 +756,14 @@ describe("Schnelleinsteig", () => {
...
-      Action.maneuver({ origin: "liverpool", destination: "north atlantic" }),
-      Action.maneuver({ origin: "london", destination: "english channel" }),
-      Action.rondel({ nation: Nation.GE, cost: 0, slot: "maneuver2" }),
-      Action.maneuver({ origin: "hamburg", destination: "north sea" }),
-      Action.maneuver({ origin: "berlin", destination: "norway" }),
-      Action.rondel({ nation: Nation.RU, cost: 0, slot: "import" }),
-      Action.import({ province: "st. petersburg" }),
-      Action.import({ province: "moscow" }),
-      Action.import({ province: "moscow" }),
-    ];
```
Comment (jreut) [inline schnelleinsteig.test.js:800]: 🙌

**Review by jreut — APPROVED** [review]
lgtm! in the spirit of emoji, 🌭

File: imperial.js (hunk 1 above)
Comment (Thrillberg) [reply imperial.js:267]: I've been caught! What do you think? Pause here and actually figure out how to do this right or let the TDD make us fix it when we need to? (I do hope to resume and finish the 𝔖𝔠𝔥𝔫𝔢𝔩𝔩𝔢𝔦𝔫𝔰𝔱𝔢𝔦𝔤 someday)

**Pull request merged**